syntax = "proto3";

package my.grpc.service;

message RequestUser {
  string name = 1;
}

message UserResponse {
  string message = 1;
  bool success = 2;
  int32 age = 3;
  int32 height = 4;
}

service MyService {
  rpc GetUserResponse (RequestUser) returns (UserResponse); // Unary RPC
  rpc StreamData (RequestUser) returns (stream UserResponse); // Server streaming
}
